apps/galaxy/src/services/utils/axios.ts

```diff
@@ -3,15 +3,36 @@ import axios from "axios";
 import { environment } from "../constants/environment";
 import { getToken } from "@src/utils/tokenService";
 
-const token = getToken() || "";
 const REQUEST_TIMEOUT = 2000;
 
+type config = {
```

Comment:
type def name better be `PascalCase`  

https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#type